src/models/github-server-app.ts

```diff
@@ -175,7 +175,9 @@ export class GitHubServerApp extends EncryptedModel {
 
 	}
 
-	static async findForUuid(uuid: string): Promise<GitHubServerApp | null> {
+	static async findForUuid(uuid: string | null): Promise<GitHubServerApp | undefined> {
```

Comment:
We are now calling this method and passing req.params.uuid from middleware where that param may not exist (cloud path) and the where clause fails when an undefined value is passed.